MAUI/Calendar/overview.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+title: About .NET MAUI Calendar control | Syncfusion
+description: Learn here the overview of Syncfusion .NET MAUI Calendar(SfCalendar) control, its basic features and functionalities of calendar.
```

Comment:
Learn here about the overview of Syncfusion .NET MAUI Calendar (SfCalendar) control, its basic features, and calendar functionalities.

Reply:
Description changed